cli/src/main/resources/help/shutdown.adoc

```diff
@@ -16,13 +15,29 @@ SYNOPSIS
 *shutdown cluster*
 
 
+USAGE CONSIDERATIONS
+--------------------
+*shutdown server*
+
+To prevent data loss, you should wait for any rebalancing operations to complete before stopping server instances.
+
+*shutdown cluster*
+
+When you gracefully stop a cluster, its state is persisted to the data folder for each node.
```

Comment:
well, `shutdown cluster` is always graceful. maybe say something like: the cluster state is persisted (hmm, only if persistence is configured) and all servers are shutdown.